Dockerfile

```diff
@@ -14,14 +14,14 @@ ENV TERM xterm
 
 ENV TERM xterm
 
+#The solr config file needs to be added after installation or it fails.
+ADD docker_files/solr /etc/default/solr.docker
```

Comment:
Yes, during the "apt-get -y install solr-server" if the /etc/default/solr file is already it gets replaced by the one from the package. So I put the file there as "/etc/default/solr.docker" and after the package has been installed move it to "/etc/default/solr".

I did spend some time trying to force apt-get to not replace the file during install, but couldn't an option that worked.